timesketch/lib/sigma_util.py

```diff
@@ -114,9 +115,12 @@ def enrich_sigma_rule_object(rule: SigmaRule, parse_yaml: bool = False):
     return parsed_rule
 
 
-def get_all_sigma_rules():
+def get_all_sigma_rules(parse_yaml: bool = False):
```

Comment:
before it was not possible to make the function parse the rules, which in the analyzer way is needed. If nothing provided, the method will use the faster way. So one has to opt in to spent CPU